docs/howto/test-json-path.md

```diff
@@ -0,0 +1,16 @@
+# test JSON Path on Dataset How to
+
+ Datasets transformed from Runs are available to be used in Reports or change detection. Manipulating Run data that has relevant data into arrays or scalar value ready for these tasks. The ability to query JSON data and test the search paths is available using the Datasets view. The view provides immediate feedback on the success or otherwise of the query.
```

Comment:
Technically speaking it's Labels that you use in Reports or CD, Datasets are only intermediate representation. So I guess you might want to try certain JSON Path to use it in Label, you won't access Dataset directly anywhere.
The second sentence is missing a verb?

Reply:
@rvansa close this